h3/src/frame/buf.rs

```diff
@@ -70,3 +70,41 @@ impl<T: Buf> Buf for BufList<T> {
         vecs
     }
 }
+
+pub struct Cursor<'a, B> {
```

Comment:
do we need an own `Cursor`? `Buf` is impl'ed for `Cursor<T:Buf>`: https://docs.rs/bytes/0.5.6/bytes/trait.Buf.html#impl-Buf-for-Cursor%3CT%3E

Reply:
Yeah, that's the problem. BufList content is not contiguous and therefore cannot implement `AsRef<[u8]>` as Cursor relies onto.